src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js

```diff
@@ -8,7 +18,7 @@ export const ListContainer = styled.div`
     background: ${colors.smoothGrey};
   }
   & > div:nth-child(n + 4) {
-    margin-top: 10px;
+    margin-top: ${spacings.spacesLv2};
```

Comment:
it's ok the name `Lv` or it's better use `Lvl` ?

Reply:
I change it to spacesLvl